README.md

```diff
@@ -133,17 +133,18 @@ steps:
-| `AZURE_TOKEN` | Azure Access Token <br> Example: `AZURE_TOKEN: $(System.AccessToken)` or `AZURE_TOKEN: $(PAT_TOKEN)` | Mandatory if  BRIDGE_COVERITY_AUTOMATION_PRCOMMENT is set true. |
+| Input Parameter                         | Description                                                                                                                                                                                                                                                                                   | Mandatory / Optional |
+|-----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| `BRIDGE_COVERITY_CONNECT_URL`           | URL for Coverity server                                                                                                                                                                                                                                                                       | Mandatory     |
+| `BRIDGE_COVERITY_CONNECT_USER_NAME`     | Username for Coverity                                                                                                                                                                                                                                                                         | Mandatory     |
+| `BRIDGE_COVERITY_CONNECT_USER_PASSWORD` | Password for Coverity                                                                                                                                                                                                                                                                         | Mandatory     |
+| `BRIDGE_COVERITY_CONNECT_PROJECT_NAME`  | Project name in Coverity                                                                                                                                                                                                                                                                      | Mandatory     |
+| `BRIDGE_COVERITY_CONNECT_STREAM_NAME`   | Stream name in Coverity                                                                                                                                                                                                                                                                       | Mandatory     |
+| `BRIDGE_COVERITY_LOCAL`                 | If enabled, Synopsys Extension will install the ‘Coverity Thick Client’ on the local system in order to execute the scan. 'coverity_install_directory' can also be given to keep the files for future scans.                                                                                   | Optional     |
```

Comment:
'coverity_install_directory' can also be given to keep the files for future scans.  - which files we are referring to?

Reply:
It will try to download coverity thick client and same files can be kept under this dir. It is nothing but caching of Coverity thick client

Reply:
'coverity_install_directory' can also be given to keep the files for future scans- can you rephrase the sentence.
Mention supported values and default value for the same.